src/cityreader/cityreader.py

```diff
@@ -1,11 +1,39 @@
+import pandas as pd
+
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, lat and lon (representing latitude and longitude).
 
 
+class City:
+
+    """
+    Base class for Cities
+
+    Attributes:
+    :var name: str
+    :var lat: float
+    :var lon: float
+    """
+
+    #  Class Variables
+
+    name = None
+    lat = None
+    lon = None
+
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
```

Comment:
Just curious about why you declare your class vars outside of the __init__ function in addition to inside?